python/fusion_engine_client/analysis/analyzer.py

```diff
@@ -1697,11 +1676,11 @@ def _plot_vehicle_data(data, time_source, is_raw=False, show_gear=False, style=N
                                           mode='markers', marker={'color': color}),
                              interval_y_axis, 1)
 
-        # Plot the vehicle speed data. If we have both corrected and uncorrected (raw) data, plot them both.
-        _plot_vehicle_data(vehicle_data, vehicle_time_source,
-                           is_raw=vehicle_measurement_type == raw_vehicle_measurement_type, show_gear=True)
-        if vehicle_measurement_type != raw_vehicle_measurement_type:
-            _plot_vehicle_data(raw_vehicle_data, vehicle_time_source, is_raw=True, show_gear=False)
+        # Plot the data. If we have both corrected and uncorrected (raw) data, plot them both.
```

Comment:
It took me a lot of reading to sort of understand this, but this comment doesn't seem accurate. If I'm understanding correctly, what this code does is:
`# Plot the data. If this data type has a separate corrected and uncorrected (raw) data message, plot them both.`

Reply:
As written it seems to imply it's referring to whether both types are present in the log.